src/test/java/com/xero/api/client/ProjectsApiTimeEntriesTest.java

```diff
@@ -104,12 +74,12 @@ public void getTimeEntriesTest() throws IOException {
         assertThat(response.getPagination().getItemCount(), is(equalTo(9)));
         assertThat(response.getPagination().getPageCount(), is(equalTo(1)));
         assertThat(response.getPagination().getPageSize(), is(equalTo(50)));
-        assertThat(response.getItems().get(0).getTimeEntryId(),is(equalTo(UUID.fromString("3cd35eca-704f-4bca-b258-236028ae8ed1"))));
-        assertThat(response.getItems().get(0).getUserId(), is(equalTo(UUID.fromString("740add2a-a703-4b8a-a670-1093919c2040"))));
-        assertThat(response.getItems().get(0).getProjectId(), is(equalTo(UUID.fromString("b021e7cb-1903-4292-b48b-5b27b4271e3e"))));
-        assertThat(response.getItems().get(0).getTaskId(), is(equalTo(UUID.fromString("7be77337-feec-4458-bb1b-dbaa5a4aafce"))));
+        assertThat(response.getItems().get(0).getTimeEntryId(),is(equalTo(UUID.fromString("00000000-0000-0000-0000-000000000000"))));
```

Comment:
why are we setting GUIDs to 00000 ?

Reply:
the values are from the openAPI, corrected the open API examples to not have zeros